tests/runfiles/linux.run

```diff
@@ -168,3 +168,9 @@ tags = ['functional', 'user_namespace']
 tests = ['groupspace_001_pos', 'groupspace_002_pos', 'groupspace_003_pos',
     'userquota_013_pos', 'userspace_003_pos']
 tags = ['functional', 'userquota']
+
+[tests/functional/simd:Linux]
+pre =
+post =
+tests = ['simd_supported']
+tags = ['functional', 'simd']
```

Comment:
nit: can you move this up in to alphabetical order with the other test groups.

Reply:
Will do. Also missed to add the new test dir to `SUBDIRS` in `Makefile.am` and to `configure.ac`. I gues that's the reason the test fails verification.